app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -70,17 +73,30 @@ class LeanplumMetricsService(private val application: Application) : MetricsServ
 
         LeanplumActivityHelper.enableLifecycleCallbacks(application)
 
-        val installedApps = MozillaProductDetector.getInstalledMozillaProducts(application)
-
-        Leanplum.start(application, hashMapOf(
-            "default_browser" to MozillaProductDetector.getMozillaBrowserDefault(application).orEmpty(),
-            "fennec_installed" to installedApps.contains(MozillaProductDetector.MozillaProducts.FIREFOX.productName),
-            "focus_installed" to installedApps.contains(MozillaProductDetector.MozillaProducts.FOCUS.productName),
-            "klar_installed" to installedApps.contains(MozillaProductDetector.MozillaProducts.KLAR.productName),
-            "fxa_signed_in" to application.settings().fxaSignedIn,
-            "fxa_has_synced_items" to application.settings().fxaHasSyncedItems,
-            "search_widget_installed" to application.settings().searchWidgetInstalled
-        ))
+        Leanplum.start(application)
+
+        MainScope().launch(Dispatchers.IO) {
+            val installedApps = MozillaProductDetector.getInstalledMozillaProducts(application)
+
+            Leanplum.setUserAttributes(
+                hashMapOf(
+                    "default_browser" to
+                            MozillaProductDetector.getMozillaBrowserDefault(application).orEmpty(),
```

Comment:
This PR doesn't remove the redundant computation in each of the `MozillaProductDetector` method calls, which are wasting the users battery & clogging up our background threads.